README.md

```diff
@@ -49,20 +49,18 @@ author.save(function(err){
     res.redirect("authors")
   }
 })
-```
-
 ```rb
-
+@author = Author.new
+@author = Author.create(author_params)
```

Comment:
`.create` = `.new` + `.save`. Because we're already using `.new`, we just need to `.save` now.